### PR TITLE
change: a few cleanups

### PR DESCRIPTION
**Pull request by eddieantonio**
Tidying up a few things. Note, I renamed the file to `gunicorn.config.py` so that it's the defaul path that Gunicorn checks. See: https://docs.gunicorn.org/en/stable/configure.html#configuration-overview

I setup a symlink because I'm pretty sure the unit file tries to lookup the non-standard config, so that must be fixed, and THEN the symlink can be safely deleted.

### CODE REVIEW

File: gunicorn.config.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+bind = "0.0.0.0:2020"
```
Comment (andrewdotn) [inline gunicorn.config.py]:
```suggestion
bind = "127.0.0.1:2020"
```

We seemed to need the more general thing in order for some stuff running inside docker to be accessible from outside that container, but this is running on the same host as the nginx server so localhost should be fine.

File: README.md [hunk 2]
```diff
@@ -17,12 +17,5 @@ ALTLab's deployment server. Enables web application redeployment via HTTPS.
 * [How to register an application with the API](./registration.md)
 * [How to configure the servers](./docs/server-config.md)
 
-## Notes
-
-* The docker service should restart all running containers when restarted. See <https://stackoverflow.com/a/18797089/6626414>.
-* You can monitor the logs in `deploy.gunicorn`:
-  - `sudo journalctl -lfu deploy.gunicorn`
-  - `sudo journalctl -lu deploy.gunicorn --since='15m'` (last 15 minutes)
```
Comment (andrewdotn) [inline README.md:25]: If the unit name hasn’t changed and these monitoring commands still work and are relevant, we should leave them here in the README.

File: gunicorn.config.py (hunk 1 above)
Comment (eddieantonio) [reply gunicorn.config.py]: Good catch!

**Review by nienna73 — APPROVED** [review]
I think these changes look good.

File: README.md (hunk 2 above)
Comment (eddieantonio) [reply README.md:25]: The unit file name has changed, but the docs are still relevant. Updated!

**Pull request merged**